packages/react-router-dom/dom.ts

```diff
@@ -176,34 +177,21 @@ export function getFormSubmissionInfo(
   payload: any;
 } {
   let method: string;
-  let action: string | null = null;
+  let action: string | null;
   let encType: string | null;
   let formData: FormData | undefined = undefined;
   let payload: unknown = undefined;
 
   if (isFormElement(target)) {
-    let submissionTrigger: HTMLButtonElement | HTMLInputElement = (
-      options as any
-    ).submissionTrigger;
```

Comment:
This was no longer being used - ever since we started sending from the `submitter` property on the event